pkg/splunk/splunk.go

```diff
@@ -72,7 +73,7 @@ func Send(customTelemetryData *telemetry.CustomData, logCollector *log.Collector
 		// OR Failure run and we do not want to send the logs
 		err := tryPostMessages(telemetryData, []log.Message{})
 		if err != nil {
-			log.Entry().WithError(err).WithField("module", "logger/splunk").Warn("Error while sending logs")
+			return errors.Wrapf(err, "Error while sending logs: %v", err)
```

Comment:
This way you would append the error twice: `Error while sending logs: connection closed: connection closed`

```suggestion
			return errors.Wrap(err, "error while sending logs")
```